AudioDeviceExample/AudioDevices/ExampleCoreAudioDevice.m

```diff
@@ -2,7 +2,7 @@
 //  ExampleCoreAudioDevice.m
 //  AudioDeviceExample
 //
-//  Copyright © 2018 Twilio, Inc. All rights reserved.
+//  Copyright © 2018-2019 Twilio, Inc. All rights reserved.
```

Comment:
This class has iOS 9 specific code for interruption handling which can now be removed.

Reply:
Removed!